mbta/update_gtfs.sh

```diff
@@ -5,4 +5,4 @@ set -e
 # otherwise realtime alerts won't work: https://github.com/mbta/OpenTripPlanner/pull/8
 
 wget -N https://mbta-gtfs-s3.s3.amazonaws.com/google_transit.zip -O var/graphs/mbta/1_MBTA_GTFS.zip
-wget -N http://www.massport.com/media/4109/massport_gtfs.zip -O var/graphs/mbta/2_loganexpress-ma-us.zip
+wget -N https://transitfeeds.com/p/massport/1137/20200328/download -O var/graphs/mbta/2_loganexpress-ma-us.zip
```

Comment:
I re-uploaded the file that was there to our own S3 bucket: https://mbta-gtfs-s3.s3.amazonaws.com/loganexpress_temp_gtfs.zip . Can you update to that link for now, while we wait for Massport to fix their feed?